challenges/objects-arrays.js

```diff
@@ -59,12 +91,25 @@ Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
-console.log(contactInfo);
+
+for(let i = 0; i < graduates.length; i++) {
+  contactInfo.push(graduates[i].first_name, graduates[i].email);
+}
+
```

Comment:
I think this will work fine -- you at least get the concept! What I would do is encapsulate the first name and the email into a string or its own object so that those pieces of information stay together. I think by the way you have it here, index 0 would be the first name of the first grad in the list and then the 1 index would be the email of the first graduate in the list. 

```
contactInfo.push({first_name: graduates[i].name, email: graduates[i].email})

OR

contactInfo.push(`${graduates[i].first_name ${graduates[i].email}`)

```

Again I think your way works to a point, but I don't think it's the best way to present the data.